app/src/main/java/com/aws/amazonlocation/ui/main/explore/ExploreViewModel.kt

```diff
@@ -540,87 +525,186 @@ class ExploreViewModel
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_ken),
-                    description = context.getString(R.string.description_ken),
-                    countryCode = context.getString(R.string.flag_ken),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_mar),
-                    description = context.getString(R.string.description_mar),
-                    countryCode = context.getString(R.string.flag_mar),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_rus),
-                    description = context.getString(R.string.description_rus),
-                    countryCode = context.getString(R.string.flag_rus),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_sdn),
-                    description = context.getString(R.string.description_sdn),
-                    countryCode = context.getString(R.string.flag_sdn),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_srb),
-                    description = context.getString(R.string.description_srb),
-                    countryCode = context.getString(R.string.flag_srb),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_sur),
-                    description = context.getString(R.string.description_sur),
-                    countryCode = context.getString(R.string.flag_sur),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_syr),
-                    description = context.getString(R.string.description_syr),
-                    countryCode = context.getString(R.string.flag_syr),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_tur),
-                    description = context.getString(R.string.description_tur),
-                    countryCode = context.getString(R.string.flag_tur),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_tza),
-                    description = context.getString(R.string.description_tza),
-                    countryCode = context.getString(R.string.flag_tza),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_ury),
-                    description = context.getString(R.string.description_ury),
-                    countryCode = context.getString(R.string.flag_ury),
-            ),
-            PoliticalData(
-                countryName = context.getString(R.string.label_vnm),
-                    description = context.getString(R.string.description_vnm),
-                    countryCode = context.getString(R.string.flag_vnm),
+        fun setPoliticalListData(context: Context) {
```

Comment:
This code looks like it's duplicated between ExploreView and MapStyleView, can it be shared between them?

Reply:
Optimized